openfisca_web_api_preview/loader/__init__.py

```diff
@@ -13,11 +13,17 @@ def extract_description(items):
         }
 
 
-def build_data(country_package_name):
+def build_data(country_package_name, extensions):
     tax_benefit_system = build_tax_benefit_system(country_package_name)
     country_package_metadata = tax_benefit_system.get_package_metadata()
+
+    if extensions is not None:
```

Comment:
Any reason not to use [this common function](https://github.com/openfisca/openfisca-core/blob/master/openfisca_core/scripts/__init__.py#L27) ?